### ISSUE TITLE
Wordnet : synset not found for many existing sense keys

### ISSUE THREAD

**Issue by rj-7** [nltk/nltk#2219]
The function nltk.corpus.wordnet.lemma_from_key(key) does not return synsets for many valid sensekeys.
`print wn.lemma_from_key(hot%3:00:00:tasty:00).synset()`
prints out 
`WordNetError: No synset found for key 'hot%3:00:00:tasty:00'`
There are many similar keys resulting the same outcome.
Also I am using WordNet 3.1 princeton web search to verify the validity of keys. But nltk has a lower wordnet version (3.0). I wonder if this is causing the problem. If I am right can we get the 3.1 upgrade on nltk soon? Desperately need this for research project.
Some other keys are:
need%2:37:00::
fair%3:00:00:beautiful:00
clarify%2:30:00::
previous%5:00:00:preceding(a):00
old%3:00:00:familiar:00
And so on...

**Comment by savkov** [comment]
I have the same problem. There are a ton of them. Just sample below.
```
reduced%3:00:04::
full%5:00:01:whole:00
uninterrupted%5:00:00:unbroken:02
minimized%5:00:02:decreased:00
spaced%5:00:00:distributed:00
saddle%2:35:10::
coin%1:21:02::
free%5:00:02:unoccupied:00
earnestly%4:02:02::
correct%5:00:00:proper:00
active%5:00:00:involved:00
completely%4:02:04::
unfortunate%3:00:04::
ever%4:02:04::
fit%5:00:00:ready:00
unfortunate%3:00:04::
before%4:02:03::
personal%5:00:00:subjective:00
conditional%5:00:00:qualified:02
take_care%2:41:12::
earn%2:40:03::
half%5:00:00:fractional:00
unfortunate%3:00:04::
ever%4:02:04::
correct%5:00:00:proper:00
live%2:42:08::
second%5:00:00:ordinal:00
```

**Comment by udnaan** [comment]
TLDR: Replace `%3:` with `%5:` to make it work.

Long version:

In wordnet, there are two types of adv. Normal and satellite. The first one uses %3 and the second one uses %5. This granularity is usually not present when converting from elsewhere.

Also, in some other cases, you may wanna normalize the sensekey by removing parenthesis e.g in the case of `previous%5:00:00:preceding(a):00` => `previous%5:00:00:preceding:00`

Moreover, if you try to verify it through the online [version](http://wordnetweb.princeton.edu/perl/webwn), note that the online version is misleading and not in sync with the actual `index.sense` file

Ref: https://wordnet.princeton.edu/documentation/senseidx5wn

**Comment by ekaf** [comment]
Fixed in #2988.
